cmd/controller/cmd/root.go

```diff
@@ -7,6 +7,10 @@ import (
 	"os"
 	"runtime"
 
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Can we move these so that they're grouped alphabetically like all of them below.

Reply:
fixed